lm_eval/tasks/ja/__init__.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+class MecabTokenizer:
+    def __init__(self) -> None:
+        from fugashi import Tagger
+        self.tagger = Tagger('-Owakati')
+
+    def normalize_answer(self, text):
+        """Lower text and remove punctuation, articles and extra whitespace."""
```

Comment:
```suggestion
        """Lower case text, remove punctuation and extra whitespace, etc."""
```

You don't seem to be removing articles?